### DISCUSSION TITLE
KES error: certificate signed by unknown authority

### DISCUSSION THREAD

**Discussion by jpchev** [minio/minio#14630] [Q&A] [answered]
after upgrading to the latest KES version i get the following error when trying to create a key with kes
```
export KES_SERVER=https://frontal_bastion_vm:7373
export KES_CLIENT_CERT=/home/centos/kes/app.cert
export KES_CLIENT_KEY=/home/centos/kes/app.key
kes key create my-app-key
```
_Error: failed to create key "my-app-key": Post "https://frontal_bastion_vm:7373/v1/key/create/my-app-key": x509: certificate signed by unknown authority_

The KES server certificate has been created with the command
`kes identity new --key kes/kes_server.key --cert kes/kes_server.crt --ip "127.0.0.1" --dns frontal_bastion_vm frontal_bastion_vm`
do you have an idea?

**Comment by harshavardhana** [comment]
> _Error: failed to create key "my-app-key": Post "https://frontal_bastion_vm:7373/v1/key/create/my-app-key": x509: certificate signed by unknown authority_

Your fronta_bastion_vm is not trusted by kes client - set the CAPATH @jpchev

**Reply by jpchev** [reply]
can you please point me to the KES doc mentionning this parameter? I can't find it

**Reply by jpchev** [reply]
@harshavardhana do you mean MINIO_KMS_KES_CAPATH ? That is for minio, not KES

**Comment by jpchev** [comment, marked as answer]
I solved using the option -k to skip TLS certificate validation
